### PR TITLE
added OR as !decide delimiter

### PR DESCRIPTION
**Pull request by lykranian**
capital OR delimiter for decisions.rb !decide plugin

also bumped version

### CODE REVIEW

File: plugins/decisions.rb [hunk 1]
```diff
@@ -24,7 +24,7 @@ def match?(cmd)
 	match /decide (.+)/, method: :decide, strip_colors: true
 
 	def decide(m, query)
-		choices = query.split(/ or|\|\| /).map(&:strip).map(&:downcase).uniq
+		choices = query.split(/ OR| or|\|\| /).map(&:strip).map(&:downcase).uniq
```
Comment (woodruffw) [inline plugins/decisions.rb]: Just do `/ or|\|\| /i` for case insensitivity.

Comment (lykranian) [inline plugins/decisions.rb]: I dislike the idea of Or and oR, but I'll change if you want it.

Comment (woodruffw) [inline plugins/decisions.rb]: Good point. Only problem then is that there's an extra space between `OR|or`.

File: plugins/decisions.rb [hunk 2]
```diff
@@ -24,7 +24,7 @@ def match?(cmd)
 	match /decide (.+)/, method: :decide, strip_colors: true
 
 	def decide(m, query)
-		choices = query.split(/ OR| or|\|\| /).map(&:strip).map(&:downcase).uniq
+		choices = query.split(/ OR|or|\|\| /).map(&:strip).map(&:downcase).uniq
```
Comment (lykranian) [inline plugins/decisions.rb]: there?

Comment (woodruffw) [inline plugins/decisions.rb]: Yep.

**Pull request merged**